src/Symfony/Component/Config/Loader/FileLoader.php

```diff
@@ -136,7 +136,11 @@ private function doImport(mixed $resource, string $type = null, bool $ignoreErro
         try {
             $loader = $this->resolve($resource, $type);
 
-            if ($loader instanceof self && null !== $this->currentDir) {
+            if (!$loader instanceof self) {
+                return $loader->load($resource, $type);
```

Comment:
@derrabus That does not work as the PSR-4 loader uses the locator but without the current dir, so the resource cannot be found. Or am I missing something?